### DISCUSSION TITLE
Place command in buffer at end of function?

### DISCUSSION THREAD

**Discussion by typecasto** [fish-shell/fish-shell#8972] [Q&A]
I've got a command I use that bumps the version number of some software I'm writing. After I run that command, 90% of the time, I want to run `git commit -m "[v$version] some_message"`. Is it possible to have a function insert `git commit -m "[v1.6.2] ` into the next command buffer, so that I can type my message or edit the command and then run it?

**Comment by faho** [comment]
It is not possible to modify the commandline from a function, unless you run it from a key binding, no.

**Comment by ridiculousfish** [comment]
This should be possible now with the fix for #8807. `commandline git commit -m "[v1.6.2]' will do it.
